### PR TITLE
Marsmedia: Change bid params from zone to zoneId

### PR DESCRIPTION
**Pull request by vladi-mmg**
According for our [Doc](https://docs.prebid.org/dev-docs/bidders/marsmedia), the bid params right now is zone at prebid server, and zoneId at our prebid js.
We fix it that the param will same.

### CODE REVIEW

File: static/bidder-params/marsmedia.json [hunk 1]
```diff
@@ -4,10 +4,10 @@
...
   "type": "object",
   "properties": {
-    "zone": {
+    "zoneId": {
       "type": "string",
       "description": "Zone ID to use."
     }
   },
-  "required": ["zone"]
+  "required": ["zoneId"]
```
[the remark this answers is not in the document]
Comment (hhhjort) [reply static/bidder-params/marsmedia.json:12]: One last thing, this applies just to the JSON schema parsing. Your adapter is free to return an error if `zoneId` is missing. Adapter errors do not block the entire request from going forward.

**Comment by mansinahar** [comment]
@vladi-mmg Friendly ping about the above comments

File: static/bidder-params/marsmedia.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (vladi-mmg) [reply static/bidder-params/marsmedia.json]: Thank you, I remove the required.

**Comment by vladi-mmg** [comment]
> @vladi-mmg Friendly ping about the above comments

Hi, I am sorry, I was at long vacation :)

**Comment by mansinahar** [comment]
@vladi-mmg No worries! Thanks for addressing the comments. We will re-review.

**Review by hhhjort — APPROVED** [review]
LGTM

**Pull request merged**